javascripts/content_scripts/privly.js

```diff
@@ -296,17 +295,16 @@ var privly = {
      * @param {array} elements array of "a" elements.
      * @return {array} Elements not updated by the function.
      */
-    moveFromBody: function(elements) {
+    moveFromBody: function (elements) {
       "use strict";
       var notUpdated = [];
       elements.forEach(
-        function(a){
+        function (a) {
           // Optimization
-          if (  a.textContent.indexOf("privlyInject1") <= 0 
-            || ! privly.correctIndirection.testAndCopyOver(a, a.textContent) ) {
+          if (a.textContent.indexOf("privlyInject1") <= 0 || !privly.correctIndirection.testAndCopyOver(a, a.textContent)) {
             notUpdated.push(a);
           }
-      });
```

Comment:
This seems like a strange change, since it is actually closing out the parenthesis from two spaces to the left.